project/dependencies.scala

```diff
@@ -10,7 +10,7 @@ object Versions {
   val bloopVersion = "1.4.8-26-cbddb8ba"
   val zincVersion = "1.4.0-M12"
   val intellijVersion = "212.4486"
-  val bspVersion = "2.0.0-M12+27-4994bd9d-SNAPSHOT"
+  val bspVersion = "2.0.0-M14+7-56ef5d5f-SNAPSHOT"
```

Comment:
We can use M14 without the latest snapshot

Reply:
```suggestion
  val bspVersion = "2.0.0-M14"
```